.github/actions/setup_test_project/setup_test_project.mjs

```diff
@@ -18,26 +17,5 @@ console.log({
 
 core.setOutput('test_project_path', test_project_path)
 
-// See https://github.com/actions/toolkit/tree/main/packages/io#cpmv.
-console.log('Copying test project fixture')
-await io.cp(
-  './__fixtures__/test-project',
-  test_project_path,
-  {
-    recursive: true,
-    copySourceDirectory: false
-  }
-)
-
-await exec(`ls ${test_project_path}`)
-
-console.log('Project deps')
-await exec(`yarn project:deps ${test_project_path}`)
-
-console.log('Install')
-await exec('yarn install', null, {
-  cwd: test_project_path,
-})
 
-console.log('Project copy')
-await exec(`yarn project:copy ${test_project_path}`)
+await exec(`yarn build:test-project --ts --link ${test_project_path}`)
```

Comment:
I suggest we disable Telemetry for this step as it will add overhead and it's not the data we want to capture:
`REDWOOD_DISABLE_TELEMETRY=1 yarn build:test-project ...`

Reply:
Done, but I disabled it in the workflow rather than in this script because it was failing otherwise